policyengine_us/variables/gov/states/va/tax/income/va_agi.py

```diff
@@ -8,5 +8,15 @@ class va_agi(Variable):
     unit = USD
     definition_period = YEAR
     reference = (
-        "https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/"
+        "https://www.tax.virginia.gov/sites/default/files/taxforms/individual-income-tax/2022/760-2022.pdf",
+        "https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/",
     )
+    defined_for = StateCode.VA
+
+    adds = ["adjusted_gross_income", "va_additions"]
+
+    subtracts = [
+        "va_subtractions",
+        "va_age_deduction",
+        "taxable_social_security",
```

Comment:
why not put in subtractions.yaml? it's separate on the form but included with other subtractions in the legal code 58.1-322.02(3)